### PR TITLE
ST mapping file generation for commonSVT resources

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 [hunk 1]
```diff
@@ -316,11 +455,144 @@ class AzSKADOServiceMapping: CommandBase
         }
 
         #Removing duplicate entries of the tuple (variableGroupId,serviceId)
-        $variableGroupSTMapping.data = $variableGroupSTMapping.data | Sort-Object -Unique variableGroupID,serviceID
+        if ($this.MappingType -eq "All" -or $this.MappingType -eq "SecureFile") {
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:458]: vargrp

Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1]: changed...

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 [hunk 2]
```diff
@@ -286,27 +372,80 @@ class AzSKADOServiceMapping: CommandBase
...
+                    Write-Progress -Activity 'Variable group/secure file mappings via build...' -CurrentOperation $bldDef.Name -PercentComplete (($counter / $buildDefnsObj.count) * 100)
 
                     $buildObj = [WebRequestHelper]::InvokeGetWebRequest($bldDef.url.split('?')[0]);
 
-                    if([Helpers]::CheckMember($buildObj[0],"variableGroups"))
-                    {
-                        $varGrps = $buildObj[0].variableGroups
-                        $varGrps | ForEach-Object{
+                    #getting secure files added in all the tasks.
+                    try {
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:386]: Is try catch needed here?

Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:392]: yes, we are drilling down, the objects property , checkmember is there, but let try catch be there for safety if any exceptional pipeline breaks , then it does not break loop

**Review by SaikumarRagala — APPROVED** [review]

**Review by Garima-msft — APPROVED** [review]

**Pull request merged**